docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,278 @@
+# Getting Started With SOCI Snapshotter
+This document is a guide to walk through an example of using the
+soci-snapshotter.  To get everything running we will need to do a few things :
```

Comment:
Nit: `s :/:/g`. Remove the space before the colon here, and the 23 other places where it occurs in this PR.

Refs:
* https://en.wikipedia.org/wiki/Colon_(punctuation)#Spacing
* https://english.stackexchange.com/questions/67394/spaces-around-a-colon